src/MOI_wrapper.jl

```diff
@@ -2488,3 +2488,66 @@ function MOI.get(
     end
     return get_intattrelement(model.inner, "IISQConstr", _info(model, index).row) > 0
 end
+
+###
+### Constraint attributes
+###
+
+# Gurobi constraint attributes as documented at https://www.gurobi.com/documentation/8.1/refman/linear_constraint_attribut.html
+const CONSTR_ATTR_TYPE =
+    Dict( "Lazy" => Integer, "DStart" => Real, "IISConstr" => Integer,
```

Comment:
No space after opening parentheses and before closing ones, see https://www.juliaopt.org/JuMP.jl/dev/style/#Whitespace-1

Reply:
Done in https://github.com/JuliaOpt/Gurobi.jl/pull/243/commits/bc9a732a30a5cddfd26e3372015a87b478c06bf8